widgets/modelviewer/src/modelViewerWidget.tsx

```diff
@@ -0,0 +1,22 @@
+import { ModelViewer, ModelViewerConfig, TagOverlay } from '@cc-components/modelviewer';
+import styled from 'styled-components';
+
+type ModelViewerProps = {
+  facility: string;
+  tagsOverlay?: string[] | TagOverlay[];
+  options?: ModelViewerConfig;
+};
+
+const StyledWrapper = styled.div`
+  height: calc(100vh - 276px);
```

Comment:
We cant have 100vh - 276px. Lets do something smart here

Reply:
Added styling to parent components.
Changed to height 100%